### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: content/assignments/techjobs-console/_index.md
Comment (codinglikeagirl42) [inline content/assignments/techjobs-console/_index.md]: Please update link.

File: content/assignments/techjobs-console/_index.md
Comment (codinglikeagirl42) [inline content/assignments/techjobs-console/_index.md]: Please update link

File: content/assignments/techjobs-console/submission/index.md
Comment (codinglikeagirl42) [inline content/assignments/techjobs-console/submission/index.md]: Please update link.

File: content/assignments/next-steps.md
Comment (codinglikeagirl42) [inline content/assignments/next-steps.md]:
```suggestion
originalAuthor: John Woolbright # to be set by page creator
originalAuthorGitHub: jwoolbright23 # to be set by page creator
reviewer: Kimberly Horan # to be set by the page reviewer
reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
lastEditor:  # update any time edits are made after review
lastEditorGitHub:  # update any time edits are made after review
lastMod:   # UPDATE ANY TIME CHANGES ARE MADE
```

File: content/assignments/techjobs-console/task-three/index.md
Comment (codinglikeagirl42) [inline content/assignments/techjobs-console/task-three/index.md]: Please update link

File: content/assignments/techjobs-console/_index.md
Comment (codinglikeagirl42) [inline content/assignments/techjobs-console/_index.md:7]:
```suggestion
originalAuthor: John Woolbright # to be set by page creator
originalAuthorGitHub: jwoolbright23 # to be set by page creator
```

File: content/assignments/techjobs-console/submission/index.md
Comment (codinglikeagirl42) [inline content/assignments/techjobs-console/submission/index.md:7]:
```suggestion
originalAuthor: John Woolbright # to be set by page creator
originalAuthorGitHub: jwoolbright23 # to be set by page creator
```

File: content/assignments/techjobs-console/task-three/index.md
Comment (codinglikeagirl42) [inline content/assignments/techjobs-console/task-three/index.md:7]:
```suggestion
originalAuthor: John Woolbright # to be set by page creator
originalAuthorGitHub: jwoolbright23 # to be set by page creator
```

File: content/assignments/techjobs-console/task-three/index.md
Comment (codinglikeagirl42) [inline content/assignments/techjobs-console/task-three/index.md:12]: If Sally created the page - please update lines 10 -12 accordingly

File: content/assignments/_index.md
Comment (codinglikeagirl42) [inline content/assignments/_index.md]:
```suggestion
lastEditor:  # update any time edits are made after review
lastEditorGitHub:  # update any time edits are made after review
lastMod:   # UPDATE ANY TIME CHANGES ARE MADE
```

File: content/assignments/techjobs-console/_index.md
Comment (codinglikeagirl42) [inline content/assignments/techjobs-console/_index.md:12]: If Sally created the page - please update lines 10 -12 accordingly

File: content/assignments/techjobs-console/submission/index.md
Comment (codinglikeagirl42) [inline content/assignments/techjobs-console/submission/index.md:12]: If Sally created the page - please update lines 10 -12 accordingly

**Review by codinglikeagirl42 — CHANGES_REQUESTED** [review]
Initial review - Metadata updates and link updates needed.  May need additional instructions when opening starter code depending on instructions and setup / assignment 0 review, which has not been submitted yet.

File: content/assignments/techjobs-console/_index.md
Comment (codinglikeagirl42) [reply content/assignments/techjobs-console/_index.md]: Link still doesn't work -> notice that you are using a different syntax for internal book links than Courtney and Sally.  See: https://raw.githubusercontent.com/LaunchCodeEducation/csharp-web-dev-curriculum/main/content/assignments/tech-jobs-oo/_index.md

File: content/assignments/techjobs-console/_index.md
Comment (codinglikeagirl42) [reply content/assignments/techjobs-console/_index.md]: Not sure if this link will be an issue once merged into main, as it seems to be hard coded instead of using relref, see above comment on bonus mission link and example.

File: content/assignments/techjobs-console/submission/index.md
Comment (codinglikeagirl42) [reply content/assignments/techjobs-console/submission/index.md]: Still getting 404 error.

**Review by codinglikeagirl42 — CHANGES_REQUESTED** [review]
Second review completed, please see comments.  If you have any questions please let me know.

**Review by icr8FreeCode — APPROVED** [review]
New pages changes. Did not verify links. Please ensure all links are working once merged @jwoolbright23

**Pull request merged**
